kombu/transport/SQS.py

```diff
@@ -450,6 +478,22 @@ def visibility_timeout(self):
         return (self.transport_options.get('visibility_timeout') or
                 self.default_visibility_timeout)
 
+    @cached_property
+    def predefined_queue_urls(self):
+        """Map of queue_name to queue URL.
+
+        For example:
+        {"my-queue":
+         "https://ap-southeast-2.queue.amazonaws.com/1234567890/my-queue"}
+
```

Comment:
I think that the comments from here onwards, however useful, belong to the documentation and not the docstring.